src/main/java/com/java/stack/StackDemo.java

```diff
@@ -0,0 +1,99 @@
+package com.java.stack;
+
+import java.util.Arrays;
+
+public class StackDemo {
+	private static int CAPACITY = 10;
+	private int size;
+	private int[] arr = new int[CAPACITY];
+
+	/**
+	 * Add a element to head array
+	 *
+	 */
+	public void add(int item) {
+		if (this.size == arr.length) {
+			// 1. Tao mot mang moi co kich thuoc lon hon gap 1.5 lan
+			int[] newArr = new int[(int) (this.size * 1.5)];
```

Comment:
attract `(int) (this.size * 1.5)` ra 1 biến local, đặt tên meaning, ví dụ : increaseSize

Reply:
Em add rồi ạ